hibernate-envers/src/main/java/org/hibernate/envers/internal/entities/mapper/relation/ToOneIdMapper.java

```diff
@@ -196,6 +196,9 @@ private boolean isIgnoreNotFound(
 			EntityInfo referencedEntity,
 			Map data,
 			Object primaryKey) {
+		if (referencedEntity.isAudited()) {
+			return false;
+		}
```

Comment:
Please follow our [code style](https://hibernate.org/community/contribute/intellij-idea/#code-style) guidelines.

```suggestion
		if ( referencedEntity.isAudited() ) {
			return false;
		}
```